OneCommunityGlobal/HGNRest
# Sushmitha project controller unit test

# Description
PR adds unit test for projectController.js,

## Related PRS (if any):
none
…

## Main changes explained:
- Added unit test file named: projectController.spec.js
- Markdown files explaining all the tests …
…

## How to test:
1. check into the current branch
2. do npm install
3. do npm run test projectController
4. All test cases should pass

## Screenshots or videos of changes:
![Screenshot 2024-06-01 at 11 00 58 AM](https://github.com/OneCommunityGlobal/HGNRest/assets/113482457/87c6fa54-f378-496c-a616-8a518562872c)
